_tools/lint_plugin.ts

```diff
@@ -38,5 +38,56 @@ export default {
         };
       },
     },
+    // https://docs.deno.com/runtime/contributing/style_guide/#prefer-%23-over-private-keyword
+    "exported-function-args-maximum": {
+      create(context) {
+        return {
+          ExportNamedDeclaration(node) {
+            const declaration = node.declaration;
+            if (declaration?.type !== "FunctionDeclaration") return;
+            const params = declaration.params;
+            const id = declaration.id;
+            switch (params.length) {
+              case 0:
+              case 1:
+              case 2:
+                break;
+              case 3: {
+                const param = params.at(-1)!;
+                switch (param.type) {
+                  case "Identifier":
+                    if (param.name === "options") return;
```

Comment:
I don't think the parameter has to be called `options`... Maybe it just has to be an object.